### PR TITLE
Feature/iii 3022

### PR DESCRIPTION
**Pull request by brampauwelyn**
### Changed
- removed the promise from inside the goToDashboardOnJobCompletion method since job isn't a promise this caused an error, so the redirect to the dashboard wasn't happening.

Screenshot 2019-08-02 at 20 53 49

- removed job from the tests

### CODE REVIEW

File: src/place-detail/ui/place-detail.controller.js [hunk 1]
```diff
@@ -165,12 +165,8 @@ function PlaceDetail(
     $state.go('split.footer.dashboard');
   }
 
-  /**
-   * @param {EventCrudJob} job
-   */
-  controller.goToDashboardOnJobCompletion = function(job) {
-    job.task.promise
-      .then(goToDashboard);
+  controller.goToDashboardOnJobCompletion = function() {
```
Comment (jdrieghe) [inline src/place-detail/ui/place-detail.controller.js]: I wonder if we still need this encapsulating method now or if we can straight up call `goToDashboard()` instead

Comment (bertramakers) [reply src/place-detail/ui/place-detail.controller.js]: Agreed, I think we can remove this and replace it with `goToDashboard` wherever it was used.

Comment (brampauwelyn) [reply src/place-detail/ui/place-detail.controller.js]: I agree, I removed the goToDashboard() function and added the function straight to the controller method

**Comment by jdrieghe** [comment]
Good to 🚢

**Pull request merged**